packages/paste-website/src/pages/components/display-pill-group/index.mdx

```diff
@@ -0,0 +1,232 @@
+
+## Examples
+
+### Basic Display Pill Group
+
+Display Pills are very basic in their functionality. They can be linked, or unlinked by passing an `href` prop to the Display Pill. A Display Pill Group can have a mixture of linked and unlinked Pills.
+
+When provided with an `href` the `DisplayPill` will render itself as an HTML Anchor element, and will respond to any anchor based events and accept any event handlers.
+
+<LivePreview scope={{DisplayPill, DisplayPillGroup, CalendarIcon}}>
+  {`<DisplayPillGroup aria-label="Your favorite sports:">
+  <DisplayPill
+    onFocus={() => {
+      console.log('Focused Tennis!');
+    }}
+    onBlur={() => {
+      console.log('Blurred Tennis!');
+    }}
+    href="https://google.com"
+  >
+    <CalendarIcon decorative size="sizeIcon10" />
+    Tennis
+  </DisplayPill>
+  <DisplayPill>Football</DisplayPill>
+  <DisplayPill href="/">Baseball</DisplayPill>
+  <DisplayPill>Basketball</DisplayPill>
+  <DisplayPill>Soccer</DisplayPill>
+</DisplayPillGroup>`}
+</LivePreview>
+
+## Composition Notes
+
+`DisplayPillGroup` takes `DisplayPill`s as children. You should not place any other type of child component directly inside of a `DisplayPillGroup`.
+
+### When to use a Display Pill Group
+
+<DoDont>
+  <Do title="Do" body="Use Display Pills in non-editable pages to represent a collection of similar entities" />
+  <Dont title="Don't" body="Don’t use Display Pills inside of a form or when editing that collection of data." />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Only pass DisplayPill as a direct descendent of a DisplayPillGroup" />
+  <Dont
+    title="Don't"
+    body="Don’t pass DisplayPillGroup any other component type, and do not wrap Display Pills in any other component of DOM element"
+  />
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/display-pill-group - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {DisplayPillGroup, DisplayPill} from '@twilio-paste/display-pill-group';
+
+export const Basic = () => {
+  return (
+    <DisplayPillGroup aria-label="Your favorite sports:">
+      <DisplayPill
+        onFocus={() => {
+          console.log('Focused Tennis!');
+        }}
+        onBlur={() => {
+          console.log('Blurred Tennis!');
+        }}
+        href="https://google.com"
+      >
+        <CalendarIcon decorative size="sizeIcon10" />
+        Tennis
+      </DisplayPill>
+      <DisplayPill>Football</DisplayPill>
+      <DisplayPill href="/">Baseball</DisplayPill>
+      <DisplayPill>Basketball</DisplayPill>
+      <DisplayPill>Soccer</DisplayPill>
+    </DisplayPillGroup>
+  );
+};
+```
+
+#### Props
+
+##### DisplayPillGroup
+
+`DisplayPillGroup` will take any Global HTML Attribute for an HTML List element, plus the following:
+
+###### `aria-label: string`
+
+Defines a string value that labels the current element.
+
+###### `element?: string`
+
+Set a custom element name.
```

Comment:
You included the default for form pill group but not here (and the description is missing there)

Reply:
Good consistency catch